src/EPPlus/DataValidation/ExcelDataValidation.cs

```diff
@@ -236,7 +236,17 @@ private string CheckAndFixRangeAddress(string address)
             {
                 throw new FormatException("Multiple addresses may not be commaseparated, use space instead");
             }
-            address = ConvertUtil._invariantTextInfo.ToUpper(address);
+
+            var addressAndSheet = address.Split('!');
```

Comment:
Using `Split` will not like this will not work if the worksheet name contains an "!". For example: 'Sheet!'!A1. You can use the ExcelAddressBase.LocalAddress property as an option if you create an ExcelAddressBase object.

Reply:
I changed it to ignore and remove worksheet names as they are unnecesary here and likely a mistake